pkg/apis/k0s.k0sproject.io/v1beta1/clusterconfig_types.go

```diff
@@ -42,7 +42,6 @@ type ClusterSpec struct {
 	Scheduler         *SchedulerSpec         `json:"scheduler,omitempty"`
 	Storage           *StorageSpec           `json:"storage"`
 	Network           *Network               `json:"network"`
-	PodSecurityPolicy *PodSecurityPolicy     `json:"podSecurityPolicy"`
```

Comment:
What happens now if user has this configured and they start update to 1.25.0? Will they have to remove this from config yaml or is it just ignored?

Reply:
This probably needs an integration test. Thanks for reminding.

Reply:
SGTM

Reply:
I've updated the old `psp_test` to prove it works